build-info/dotnet/product/cli/release/2.1/build.semaphore

```diff
@@ -1 +1 @@
-20180605-02-1750541
+20180605-09-1764970
```

Comment:
This should trigger the publish steps again--are you trying to trigger them too or just the dependency updates?

Reply:
Good point, will revert that one